src/presentation/http/router/noteSettings.test.ts

```diff
@@ -258,75 +287,88 @@ describe('NoteSettings API', () => {
 
       expect(response?.json().message).toStrictEqual(expectedMessage);
     });
-
-    test.todo('We should to check team value in tests');
-
-    test.todo('Returns team by public id with 200 status, user is not creator of the note, but a member of the team');
-
-    test.todo('Returns 403 when user authorized, but not member of the team');
   });
 
   describe('PATCH /note-settings/:notePublicId ', () => {
-    test('Update note settings by public id with 200 status, user is in team with role write', async () => {
-      /** create test user */
+    test.each([
```

Comment:
![image](https://github.com/codex-team/notes.api/assets/59017579/da74d02f-6b06-4640-90c1-5273e00fbda3)


It's all still there, i just didn't refactor it because my refactoring is about team rights. 

But maybe in the future everything will be in test.each with params